src/components/header/Logo.tsx

```diff
@@ -51,7 +51,7 @@ export default function Logo({ branding }: { branding?: IBranding }) {
 
   return (
     <StyledButton $beta={beta} variant="ghost" asChild>
-      <AppStateLink href="/onboarding" aria-label={t("Go to home page")}>
+      <AppStateLink href="/" aria-label={t("Go to home page")}>
```

Comment:
Onboarding needs to be reworked anyway for the a11ymap, so I removed it for now.